src/Selenium2Driver.php

```diff
@@ -160,19 +135,9 @@ public function getDesiredCapabilities()
     }
 
     /**
-     * Sets the WebDriver instance
-     *
-     * @param WebDriver $webDriver An instance of the WebDriver class
-     */
-    public function setWebDriver(WebDriver $webDriver)
```

Comment:
Is this behaviour really required? To me it seems like the perfect invitation for hackeries and seriously weird bugs.